Lectures/Lecture8_OpenScienceReflections.md

```diff
@@ -1,41 +1,97 @@
-
+ To reflect on the key concepts in open science that we have discussed in this course. 
+ To practice storytelling and video production skills in creating a short (scientific) pitch.
+ 
 ## Motivation: 
-__sketch the context of the lecture, why is this topic timely or urgent to cover, and what is desired or unwanted in the status quo.__
+The last lecture of the course is an opportunity to reflect back on what has been discussed and learned, and what are the main take aways from this course. 
+The symposium format allows the participants to also create an overall impression of what standpoints their peers have developed.
 
 ## Vision of outcome
-__What skills and capabilities will the course participants obtain after finishing this lecture and the corresponding assignments?__
++ Participants choose a conscious position of how to practice open science in the next phases of our scientific career.
++ Course participants can appreciate the logical reasoning behind choosing the selection of topics that are covered in this course.
 
 ## Embedding/community
-__please mention the main community that should be considered when studying the topics of this lecture, e.g. individual researchers, research community, employers, general public, etc. __
+
+The videos in this symposium are prepared by the participants for the other participants and other potential stakeholders that can benefit from more awareness about the open science concepts.
 
 ## Methods/exercises
-**Topics covered in lecture**
-+ topic 1
-+ topic 2
+
++ Students are distributed in teams of 3-4 to select an open science concept from the list below, or a similar statement they choose, and express their conviction about this topic in a video pitch of 90 seconds. Instructions for making a successful pitch are provided in the link under the preparation section.
+
++ After watching all the videos in the class, each student distributes 100 points between the presented pitches based on how convincing they find that video.
+
+### Open science concepts
+
++ The reproducibility crisis does (not) matter because ... 
++ Fraud in science, can (not) be fixed with open science.
++ Transdisciplinary and interdisciplinary science is an oximoron (essential for solving urgent societal issues).
++ Open science practices are risky because ...
++ Open science practices are rewarding because ...
++ Opposition to open science is legitimate because ...
++ Version control is (not) essential for every type of collaborative project.
++ Creative commons licensing shoud be left to lawyers.
++ Readme files are overrated (underrated), because ...
++ Template files are restrictive (helpful), because ...
++ Community management is essential for a big collaboration.
++ Community management is not a scientific practice and should not be counted as one (e.g. in promotion).
++ Big-science collaborations are more (less) self correcting than small research groups.
```

Comment:
Add '+ Individual excellence is (not) rewarded in science anymore, because of Open Science'